### PR TITLE
can not turn off @angular-eslint/template/eqeqeq

### PR DESCRIPTION
**Pull request by gubo**
1. eqeqeq app
2. app .eslintrc.json includes "@angular-eslint/template/eqeqeq":"off"
3. app.component.html has *ngIf="ready == true"
4. nx run eqeqeq:lint still reports error

why i can not override === ? logically i might want ==

### CODE REVIEW

File: apps/eqeqeq/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        ],
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "nx-example",
+            "style": "kebab-case"
+          }
+        ],
+        "@angular-eslint/template/eqeqeq": "off"
```
Comment (puku0x) [inline apps/eqeqeq/.eslintrc.json:28]: You should add the rule for `*.html` files.

```js
    {
      "files": ["*.html"],
      "extends": ["plugin:@nrwl/nx/angular-template"],
      "rules": {
        "@angular-eslint/template/eqeqeq": "off"
      }
    }
  ]
}
```

It sounds weird but it works both separated template (`*.html`) and inline template (`*.ts`).

**Comment by gubo** [comment]
sorry, i dont follow ... i have already added the rule ... nx run
eqeqeq:lint fails ...
how do i add NX-BRANCH environment variable ? have to create a workspace
action, or ?
thanks :)

**Comment by puku0x** [comment]
@gubo Trust me, it should be successful by following `.eslintrc.json`.
```js
{
  "extends": ["../../.eslintrc.json"],
  "ignorePatterns": ["!**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "extends": [
        "plugin:@nrwl/nx/angular",
        "plugin:@angular-eslint/template/process-inline-templates"
      ],
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          {
            "type": "attribute",
            "prefix": "nxExample",
            "style": "camelCase"
          }
        ],
        "@angular-eslint/component-selector": [
          "error",
          {
            "type": "element",
            "prefix": "nx-example",
            "style": "kebab-case"
          }
        ]
      }
    },
    {
      "files": ["*.html"],
      "extends": ["plugin:@nrwl/nx/angular-template"],
      "rules": {
        "@angular-eslint/template/eqeqeq": "off"
      }
    }
  ]
}
```

Pull Request is not the best place to provide a repro, though.

**Comment by gubo** [comment]
ah i see, i was trying to add it only to *.ts ... i put it in for *.html
and it is working correctly ...
thanks for your help
arigato gozaimasu :)

**Comment by FrozenPandaz** [comment]
Looks like this is resolved now.

Thank you all!

File: apps/eqeqeq/.eslintrc.json (hunk 1 above)
Comment (vikramkln) [reply apps/eqeqeq/.eslintrc.json:28]: yes it working now for me

**Pull request closed without merging**